### PR TITLE
Represent season as Discord webhook content

### LINKED ISSUE #32: Add top scorers to daily webhook
In every daily webhook from LC-Dailies, we need to include a version of the current season's leaderboard. However, on Sundays when the webhook is triggered, the leaderboard should display the results of the previous week. This is because there are no submissions for the new season at that point during the Sunday daily webhook.

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Resolves #32.

### CODE REVIEW

File: lib/leaderboard/leaderboard_client.ts [hunk 1]
```diff
@@ -121,3 +121,5 @@ export interface LeaderboardClient {
    */
   listSeasons(): Promise<Season[]>;
 }
+
+// TODO: Calculate the top scorers of a given season to eventually be formatted in a leaderboard table.
```
Comment (EthanThatOneKid) [inline lib/leaderboard/leaderboard_client.ts:125]: Resolve TODO comment.

File: api/dailies.ts [hunk 2]
```diff
@@ -17,37 +19,90 @@ export function makeDailyWebhookPostHandler(
...
     `**Snack**: Here is a snack to get your brain working: ${snacks.pickRandom()}`,
     "",
     "Submit your solution by typing `/lc submit YOUR_SUBMISSION_URL` below! ([more info](https://acmcsuf.com/lc-dailies-handbook))",
   ].join("\n");
+
+  if (!options.season) {
+    return content;
+  }
+
+  // TODO: Render the leaderboard.
```
Comment (EthanThatOneKid) [inline api/dailies.ts:106]: Resolve TODO comment.

**Review by EthanThatOneKid — COMMENTED** [review]
Resolve TODO comments.

File: lib/leaderboard/leaderboard_client.ts (hunk 1 above)
Comment (EthanThatOneKid) [reply lib/leaderboard/leaderboard_client.ts]: Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/33/commits/186abd9c5005d861ae188bef586828a908986cb9>.

File: api/dailies.ts (hunk 2 above)
Comment (EthanThatOneKid) [reply api/dailies.ts]: Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/33/commits/4827940a2de731296b3b38d0affdbfeda8338b30>.

**Review by EthanThatOneKid — COMMENTED** [review]
LGTM 😎

**Review by karnikaavelumani — APPROVED** [review]
Your code looks very clean. Thank you for the hard work!

**Pull request merged**